java/serve/src/main/java/io/ray/serve/config/ReplicaConfig.java

```diff
@@ -142,10 +142,14 @@ public static ReplicaConfig fromProto(io.ray.serve.generated.ReplicaConfig proto
     if (proto == null) {
       return null;
     }
+    Object[] initArgs = null;
+    if (0 != proto.getInitArgs().toByteArray().length) {
+      initArgs = MessagePackSerializer.decode(proto.getInitArgs().toByteArray(), null);
```

Comment:
Why do we need  change this line?

Reply:
If `proto.getInitArgs().toByteArray()` is empty array,  `MessagePackSerializer.decode` will throw exception.